src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -282,6 +282,25 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     executeOrResume(backendCall, runIdForResumption = runId)
   }
 
+  def useCachedCall(cachedCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
+    val log = workflowLoggerWithCall(backendCall)
+    Try(ioInterface(backendCall.workflowDescriptor.workflowOptions).copy(cachedCall.callGcsPath, backendCall.callGcsPath)) match {
```

Comment:
It'd be better to use `authenticatedAsUser(backendCall.workflowDescriptor) { interface => interface.copy(...) ... }` here instead of calling `ioInterface`, because it creates a new instance of `GoogleCloudStorage`.
The idea of IoInterface was primarily to be used only by engine functions, while backends still have the ability to use classes with more advanced functionalities that engine function wouldn't need.
I realize this is not clear, it's a bit better in the refactored version because IoInterface is provided by the file system, not the backend so it's less confusing.